Automattic/jetpack
# Auto-labeling: prefix Jetpack module names with [Feature]

> **Warning**
> ~Before we merge this, we'll need to update the existing labels to match this new structure. If we do not, new labels will be automatically created **on top** of the existing ones. We do not want that.~ This has now been done.

## Proposed changes:

In order to standardize labels across tools and repos, let's prefix our different module labels with "[Feature]".

### Other information:


## Jetpack product discussion

* pciE2j-2ce-p2
* p1690467214890739-slack-CQD1HH4MA

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

This cannot be tested in this repo. You can, however, test it in a fork where you would merge this PR to `trunk`, and then try to open a PR that touches module files.
See https://github.com/jeherve/jetpack/pull/96 for an example.
